src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -66,9 +64,17 @@ const ImportMediaView = ({ prevView, closeModal, nextView }: Props) => {
     if (projectWithMedia.mediaFilePath == null) {
       return;
     }
-    setCurrentProject(projectWithMedia);
+    setCurrentProject(projectWithMedia as RuntimeProject);
     nextView();
-  };
+  }, [currentProject, mediaFilePath, nextView, setCurrentProject]);
+
+  useKeypress(handleTranscribe, isAwaitingMedia, ['Enter', 'NumpadEnter']);
+
+  if (currentProject === null) {
+    return null;
+  }
+
+  const projectName = currentProject.name;
```

Comment:
Having a look at the code, this variable (which gets redefined on every render) is only used once. I reckon it might be better for readability to just change the single usage from `projectName` to `currentProject.name` and delete the variable.

Reply:
Change made here: [a925e32](https://github.com/chloebrett/mlvet/pull/175/commits/a925e3217cf3cc53933cfd6ec287d1e8814190b2)